app/lib/submission_builder/ty2022/states/ny/individual_return.rb

```diff
@@ -64,13 +66,15 @@ def documents_wrapper
                 xml.SCHOOL_CD @submission.data_source.school_district_number
                 xml.SCHOOL_NAME @submission.data_source.school_district&.truncate(30)
                 xml.PR_EMP_DESC @submission.data_source.direct_file_data.primary_occupation
-                xml.COUNTRY_NAME @submission.data_source.mailing_country
+                #xml.COUNTRY_NAME @submission.data_source.mailing_country
```

Comment:
A comment to explain why country is not needed might help future maintainers

Reply:
Good call!